openfl/transport/grpc/aggregator_server.py

```diff
@@ -54,6 +54,19 @@ def __init__(
         self.root_certificate = root_certificate
         self.certificate = certificate
         self.private_key = private_key
+
+        if hasattr(self.aggregator, "is_connector_available"):
+            self.use_connector = self.aggregator.is_connector_available()
+        else:
+            self.use_connector = False
+
+        if self.use_connector:
+            self.local_grpc_client = (
+                self.aggregator.get_local_grpc_client()
+            )  # Initialize the local gRPC client
+        else:
+            self.local_grpc_client = None
```

Comment:
To complete the renaming, I suggest calling those resp. `interop_client` and `get_interop_client()`